### ISSUE TITLE
Quiescence bind mounted file failure

### ISSUE THREAD

**Issue by skupkosk** [vmware/open-vm-tools#347]
When running snapshot with "Quiece guest FS" enabled, the snapshot task fail on freezing bind mounted file.
This is implemented for chrooted systemd services with notify type as follows:
```
ExecStartPre=-/bin/mkdir -p /chroot/%i/run/systemd ; \
    /usr/bin/touch /chroot/%i/run/systemd/notify ; \
    /bin/mount -o bind,rw /run/systemd/notify /chroot/%i/run/systemd/notify
```
Simple check whether the mountpoint is directory could solve it.

Debug logs entries:
```
[Jun 21 12:41:12.350] [   debug] [vmbackup] *** VmBackupSyncDriverStart
[Jun 21 12:41:12.350] [   debug] [vmsvc] SyncDriver: Skipping remote filesystem, name=systemd-1, mntpt=/proc/sys/fs/binfmt_misc.
[Jun 21 12:41:12.350] [   debug] [vmsvc] SyncDriver: Calling backend 0.
[Jun 21 12:41:12.350] [   debug] [vmsvc] SyncDriver: Freezing using Linux ioctls...
[Jun 21 12:41:12.350] [   debug] [vmsvc] SyncDriver: opening path '/chroot/lin/etc/rsyslog.d'.
[Jun 21 12:41:12.350] [   debug] [vmsvc] SyncDriver: freezing path '/chroot/lin/etc/rsyslog.d' (fd=8).
[Jun 21 12:41:12.396] [   debug] [vmsvc] SyncDriver: successfully froze '/chroot/lin/etc/rsyslog.d' (fd=8).
[Jun 21 12:41:12.396] [   debug] [vmsvc] SyncDriver: opening path '/run/rsyslog-lin/systemd/notify'.
[Jun 21 12:41:12.396] [   debug] [vmsvc] SyncDriver: failed to open '/run/rsyslog-lin/systemd/notify': 6 (No such device or address)
[Jun 21 12:41:12.396] [   debug] [vmsvc] SyncDriver: Thawing fd=8.
[Jun 21 12:41:12.396] [   debug] [vmsvc] SyncDriver: Closing fd=8.
[Jun 21 12:41:12.396] [ warning] [vmbackup] Error freezing filesystems.
[Jun 21 12:41:13.351] [   debug] [vmbackup] *** VmBackupAsyncCallback
[Jun 21 12:41:13.351] [   debug] [vmbackup] *** VmBackupPostProcessCurrentOp
[Jun 21 12:41:13.351] [   debug] [vmbackup] Async request 'VmBackupSyncDriverStart' completed
[Jun 21 12:41:13.351] [   debug] [vmbackup] *** VmBackupSyncDriverReadyForSnapshot
[Jun 21 12:41:13.351] [   debug] [vmbackup] *** VmBackupEnableSync
[Jun 21 12:41:13.351] [    info] [guestinfo] New value for stats-interval is 20s.
[Jun 21 12:41:13.351] [    info] [guestinfo] New value for poll-interval is 30s.
[Jun 21 12:41:13.351] [   debug] [vmsvc] Flushed 20 log messages from cache after resuming log IO.
[Jun 21 12:41:13.351] [   debug] [vmbackup] *** VmBackup_SendEvent
[Jun 21 12:41:13.351] [   debug] [vmbackup] Sending vmbackup event: vmbackup.eventSet req.error 3 Error when enabling the sync provider.
[Jun 21 12:41:13.351] [   debug] [vmsvc] RpcChannel: Sending: 69 bytes
[Jun 21 12:41:13.351] [   debug] [vmsvc] VSockChan: Sending request for conn 9,  reqLen=69
[Jun 21 12:41:13.351] [   debug] [vmsvc] SimpleSock: Sent 101 bytes from socket 9
[Jun 21 12:41:13.354] [   debug] [vmsvc] SimpleSock: Recved 4 bytes from socket 9
[Jun 21 12:41:13.355] [   debug] [vmsvc] SimpleSock: Recved 14 bytes from socket 9
[Jun 21 12:41:13.355] [   debug] [vmsvc] VSockOut: recved 2 bytes for conn 9
[Jun 21 12:41:13.355] [   debug] [vmsvc] RpcChannel: Recved 0 bytes
[Jun 21 12:41:14.356] [   debug] [vmbackup] *** VmBackupAsyncCallback
[Jun 21 12:41:14.356] [   debug] [vmbackup] *** VmBackupPostProcessCurrentOp
[Jun 21 12:41:14.356] [   debug] [vmbackup] *** VmBackupStartScripts
[Jun 21 12:41:14.356] [   debug] [vmbackup] Trying to run scripts from /etc/vmware-tools/backupScripts.d
[Jun 21 12:41:15.357] [   debug] [vmbackup] *** VmBackupAsyncCallback
[Jun 21 12:41:15.357] [   debug] [vmbackup] *** VmBackupPostProcessCurrentOp
[Jun 21 12:41:15.357] [   debug] [vmbackup] VmBackupPostProcessCurrentOp: checking VmBackupOnFreezeFail
[Jun 21 12:41:15.357] [   debug] [vmbackup] Async request 'VmBackupOnFreezeFail' completed
[Jun 21 12:41:15.357] [   debug] [vmbackup] *** VmBackupFinalize
[Jun 21 12:41:15.357] [   debug] [vmbackup] *** VmBackup_SendEvent
[Jun 21 12:41:15.357] [   debug] [vmbackup] Sending vmbackup event: vmbackup.eventSet req.done 0
```

**Comment by skupkosk** [comment]
Running on Debian9 with open-vm-tools of version 2:10.1.5-5055683-4+deb9u1.
Upgrade of open-vm-tools to version from backports 2:10.3.10-1~bpo9+1 and adding following lines to tools.conf workarounded the issue.
```
[vmbackup]
excludedFileSystems=/chroot/*/run/systemd/*
```
Feature submitted in https://github.com/vmware/open-vm-tools/commit/477ed7490a0c8376b050b49af84a07e82490fbe6 included since 10.2.5 version.

**Comment by skupkosk** [comment]
Confirm the upgrade open-vm-tools to the backported version solved the issue and the excludedFileSystems is not needed.
